privacyidea/lib/policy.py

```diff
@@ -1409,7 +1489,8 @@ def set_policy(name=None, scope=None, action=None, realm=None, resolver=None,
     :type check_all_resolvers: bool
     :param user_case_insensitive: The username should be case-insensitive.
     :type user_case_insensitive: bool
-    :param conditions: A list of 5-tuples (section, key, comparator, value, active) of policy conditions
+    :param conditions: A list of 5- or 6-tuples (section, key, comparator, value, active, handle_missing_data) of
```

Comment:
makes me want to die. is it possible to introduce a class or dataclass to group this data together